stablehlo/dialect/TypeInference.cpp

```diff
@@ -75,6 +75,12 @@ bool allQuantized(ArrayRef<Type> typeRange) {
       typeRange, [&](Type val) { return getElementTypeOrSelf(val).isa<T>(); });
 }
 
+template <typename T>
+bool allQuantized(Type tp1, Type tp2) {
```

Comment:
thanks, thought about it.. trying to keep names neutral because we have Ops with operands `lhstype` and `rhstype`. naming consistent with HLO trait apis.